src/screens/Watch/Components/ControlBar/VolumeControl/index.js

```diff
@@ -28,6 +28,17 @@ function VolumeControl({ muted = false, volume = true, dispatch }) {
     }
   };
 
+  const sliderRef = useRef();

```

Comment:
so there isn't a super elegant way to get around `findElementById` or `$` when we want to mess with focus but useRef + useEffect with creating a function available to the entire `window` is a workaround of sorts. 

https://www.reddit.com/r/react/comments/1b5nwoh/focusing_on_elements_programmatically_in_react/ - others seem to also converge around this